pkg/processor/procbuiltin/insertfield.go

```diff
@@ -26,10 +26,9 @@ const (
 	insertFieldKeyName     = "insertfieldkey"
 	insertFieldPayloadName = "insertfieldpayload"
 
-	insertFieldConfigStaticField    = "static.field"
-	insertFieldConfigStaticValue    = "static.value"
-	insertFieldConfigTimestampField = "timestamp.field"
```

Comment:
This setting was removed completely because there is no `Record.CreatedAt` field anymore. It might make more sense to add a setting like `metadata.field` to fetch a field from metadata.